spec/controllers/pieces_controller_spec.rb

```diff
@@ -76,5 +77,56 @@
       end
     end
 
+    context "check?" do
+      it "does not return an error message if a white piece puts the black king in check" do
```

Comment:
@smiths15 I was able to check for error messages as well

Here is a good resource to see the codes for different errors: http://billpatrianakos.me/blog/2013/10/13/list-of-rails-status-code-symbols/

I've been raising the :method_not_allowed when someone tries to move a piece incorrectly. This corresponds to a 405 error. Everything should be good now!